arangod/IResearch/IResearchCommon.cpp

```diff
@@ -60,6 +60,13 @@ arangodb::LogTopic& logTopic() { return TOPIC; }
 /*static*/ std::string const StaticStrings::PrimarySortCompressionField("primarySortCompression");
 /*static*/ std::string const StaticStrings::StoredValuesField("storedValues");
 /*static*/ std::string const StaticStrings::CollectionNameField("collectionName");
+/*static*/ std::string const StaticStrings::ConsolidationIntervalMsec("consolidationIntervalMsec");
```

Comment:
https://en.wikipedia.org/wiki/Millisecond Ms better than Msec

Reply:
But we already use Msec in code and documentation. So let it be as is